gtk/gtk.go.h

```diff
@@ -376,6 +376,12 @@ toGtkLayout(void *p)
 	return (GTK_LAYOUT(p));
 }
 
+static GtkTreeModelSort *
+toGtkTreeModelSortable(void *p)
+{
+    return (GTK_TREE_MODEL_SORT(p));
+}
+
```

Comment:
that one is not needed (and not used), you have already implemented `static GtkTreeModelSort *toGtkTreeModelSort(void *p)`

Reply:
Ok will check that out.

Reply:
see #514